examples/commandline/spear_qcp/target_algorithm/scripts/genericWrapper.py

```diff
@@ -15,17 +15,17 @@
 @warning:  use "--" after the last additional argument of the wrapper to deactivate prefix matching!
 """
 
-import sys
+import json
```

Comment:
This looks like you touched the external script generic wrapper, which might make it hard to update it later. Not sure if this was intended?

Reply:
Why is it hard to update it later?

Reply:
Seems to just be a formatting change, not an implementation one. I always don't know what is meant by hard to update later.

Reply:
I mean that if someone does an update to the upstream of the generic wrapper, the change cannot be applied directly to this verison because of the different formatting. Instead, someone will have to go through the changes and manually apply them.

Reply:
That shouldn't be an issue as long as there are no current PR's that need to be merged. We did the same with Auto-sklearn without much issue as long as it's done at a good time.

Reply:
Because it's a separate repo: https://github.com/automl/GenericWrapper4AC

I don't mean PRs to SMAC, but changes to that external repo. However, the code already appears different, so I don't know what we're looking at here. Anyway, if all of you are happy, please ignore my comment.